src/components/Login.tsx

```diff
@@ -13,60 +13,93 @@ import {
     Text,
     VStack
 } from "@chakra-ui/react";
-import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons'
+import { ViewIcon, ViewOffIcon } from "@chakra-ui/icons"
 import { useNavigate } from "react-router-dom";
+import { login } from "../utils/fetchData";
+
 
 const Login = () => {
+    const [email, setEmail] = useState("")
     const [password, setPassword] = useState("");
-    const [type, setType] = useState('password');
+    const [type, setType] = useState("password");
     const [showPassword, setShowPassword] = useState(false);
 
     const navigate = useNavigate();
-    const navigateToHome = () => {
-        navigate("/home");
-    }
+
     const navigateToRegister = () => {
         navigate("/register");
     }
     const handleShowPassword = () => {
-        if (type === 'password') {
-            setType('text')
+        if (type === "password") {
+            setType("text")
             setShowPassword(true)
         } else {
-            setType('password')
+            setType("password")
             setShowPassword(false)
         }
     }
 
+    const handleLogin = () => {
+        login(email, password)
+        .then((data) => {
+            if (data.status === 200) {
+                navigate("/home");
+            }
+        })
+        .catch((error) => {
+            console.log(error);
+            if (error) {
```

Comment:
Are you going to navigate user on page 404 for any error? How are you going to handle wrong credential case?

Reply:
Error handling for other issue will be the next ticket as it is said in the description.